crates/proof-of-sql/src/sql/proof/proof_plan.rs

```diff
@@ -13,9 +13,9 @@ use core::fmt::Debug;
 #[enum_dispatch::enum_dispatch(DynProofPlan)]
 pub trait ProofPlan: Debug + Send + Sync + ProverEvaluate {
     /// Form components needed to verify and proof store into `VerificationBuilder`
-    fn verifier_evaluate<S: Scalar>(
+    fn verifier_evaluate<S: Scalar, B: VerificationBuilder<S>>(
         &self,
-        builder: &mut VerificationBuilder<S>,
+        builder: &mut B,
```

Comment:
Is there a reason we can't just do this everywhere?
```suggestion
        builder: &mut impl VerificationBuilder<S>,
```

Reply:
Suggestion applied.